packages/dd-trace/test/appsec/iast/analyzers/sql-injection-analyzer.sequelize.plugin.spec.js

```diff
@@ -0,0 +1,70 @@
+'use strict'
+
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+const { prepareTestServerForIast } = require('../utils')
+const { storage } = require('../../../../../datadog-core')
+const iastContextFunctions = require('../../../../src/appsec/iast/iast-context')
+const { newTaintedString } = require('../../../../src/appsec/iast/taint-tracking/operations')
+const vulnerabilityReporter = require('../../../../src/appsec/iast/vulnerability-reporter')
+
+describe('sql-injection-analyzer with sequelize+pg', () => {
```

Comment:
should be `sequelize+mysql` instead of `sequelize+pg`

Reply:
Yes, you're right. I started to try it in pg but finally I tested it in mysql....